tests/verifier/lib/TPP.pm

```diff
@@ -141,7 +141,7 @@ our $test_suite_tcp_pp =
                     failure => ['Segmentation fault', 'Assertion', 'ERROR']
                 },
                 client => {
-                    success => ['Test ended', 'Summary: Latency is', 'RunTime=10'],
+                    success => ['Test ended', 'Summary: Latency is'],
```

Comment:
I think that it's better to keep the check for "RunTime" string:
```suggestion
                    success => ['Test ended', 'Summary: Latency is', 'RunTime='],
```

Reply:
Will do